niworkflows/data/t1-mni_registration_testing_000.json

```diff
@@ -1,24 +1,23 @@
 {
     "dimension": 3,
     "convergence_threshold": [1e-07, 1e-08],
-    "convergence_window_size": [15, 5],
+    "convergence_window_size": [10, 5],
     "metric": ["Mattes", "Mattes"],
     "metric_weight": [1, 1],
     "radius_or_number_of_bins": [56, 56],
-    "transforms": ["Rigid", "Affine"],
-    "transform_parameters": [[1.0], [1.0]],
-    "number_of_iterations": [[20], [15]],
-    "initial_moving_transform_com": true,
+    "transforms": ["Affine", "BSplineSyN"],
```

Comment:
Was this intended to be part of this PR? Seems unrelated to initial transform setting.